core/transcoder.go

```diff
@@ -1,10 +1,15 @@
 package core
 
+//go:generate ./test_segment.sh
```

Comment:
We could also add this to the Makefile so it's automatically run every time `test_segment.sh` is modified. Same way we generate the protobufs automatically as needed. Don't need to introduce an explicit `go generate` build step. For example:

```make
all: core/testSegment.go <other files here>

core/testSegment.go: core/test_segment.sh
    core/test_segment.sh
```

(don't copy the above; make expects a leading tab and will break with leading spaces as I have. Dunno how to get tabs in here without taking the text box out of focus...)

Reply:
Fixed